### DISCUSSION TITLE
Deleted Githubaccount - name is already in use

### DISCUSSION THREAD

**Discussion by LiberteIV** [community/community#38200] [Other Feature Feedback, Questions, & Ideas]
I created a new github account and selected a unique name as this is mandatory for creating any github account, by mistake i used a wrong emailadress during the creation process, i tried to change the wrong email adress to the right one but github had no option to change the primary email adress, so the account was not verified by email because i could not access the email i miss-typed.
So i deleted this github account and tried creating a new one, now the problem is even after deleting the githubaccount with the false email, the unique name is still in use, i am trying to use the unique name for my new github account but it doesnt work as github says this name is already in use, but why is it already in use if i deleted the account which used that unique name, any help.....?

**Comment by airtower-luna** [comment]
That is to be expected, the [documentation states](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-your-personal-account/deleting-your-personal-account):

> After 90 days, the account name also becomes available to anyone else to use on a new account.

So I'm afraid you'll probably have to wait.
